packages/app-cli/package.json

```diff
@@ -34,9 +34,7 @@
     "owner": "Laurent Cozic"
   },
   "version": "2.6.1",
-  "bin": {
-    "joplin": "./main.js"
-  },
+  "bin": "./main.js",
```

Comment:
Not sure what I've changed here - should check it still works when the app is built.